ChatBot/Makefile

```diff
@@ -0,0 +1,23 @@
+run:
```

Comment:
Will this work for both python 2 and 3?

Reply:
I think we should Integrate it as part of travis build. Opinions @Raxerz @paavininanda @Buddhiprabha

Reply:
@shreyans29 what purpose shall it serve?
I have created this make file just to map larger **python** commands to shorter **make** commands.
i.e. instead of `python bot.py run`, you can call `make run`

Reply:
@Raxerz it supoorts both Python 2 and 3.

Reply:
okay.. about version, rasa has support for both, but you must have chosen one of those and moved ahead with it right? APIs for python 2 and python 3 are different. I doubt we can build this code with both.

Reply:
@shreyans29 I had used Python 3. And accordingly recommended this in the Documentation.

Reply:
yes.. I am saying that this code won't work with python 2. If you used python 3 which is a nice decision, everyone contributing in future will have to use python 3.

Reply:
Okay! Got it.